browser/go/conversion-test/conversion_test.go

```diff
@@ -0,0 +1,451 @@
+	registerOrderEventField(description, boilerplate+"makerAddress")
+	registerOrderEventField(description, boilerplate+"takerAddress")
+	registerOrderEventField(description, boilerplate+"senderAddress")
+	registerOrderEventField(description, boilerplate+"feeRecipientAddress")
+	registerOrderEventField(description, boilerplate+"exchangeAddress")
+	registerOrderEventField(description, boilerplate+"makerAssetData")
+	registerOrderEventField(description, boilerplate+"makerAssetAmount")
+	registerOrderEventField(description, boilerplate+"makerFeeAssetData")
+	registerOrderEventField(description, boilerplate+"makerFee")
+	registerOrderEventField(description, boilerplate+"takerAssetData")
+	registerOrderEventField(description, boilerplate+"takerAssetAmount")
+	registerOrderEventField(description, boilerplate+"takerFeeAssetData")
+	registerOrderEventField(description, boilerplate+"takerFee")
+	registerOrderEventField(description, boilerplate+"expirationTimeSeconds")
+	registerOrderEventField(description, boilerplate+"salt")
+}
+
+func registerSignedOrderTest(description string) {
+	registerSignedOrderField(description, "chainId")
+	registerSignedOrderField(description, "makerAddress")
+	registerSignedOrderField(description, "takerAddress")
+	registerSignedOrderField(description, "senderAddress")
+	registerSignedOrderField(description, "feeRecipientAddress")
+	registerSignedOrderField(description, "exchangeAddress")
+	registerSignedOrderField(description, "makerAssetData")
+	registerSignedOrderField(description, "makerAssetAmount")
+	registerSignedOrderField(description, "makerFeeAssetData")
+	registerSignedOrderField(description, "makerFee")
+	registerSignedOrderField(description, "takerAssetData")
+	registerSignedOrderField(description, "takerAssetAmount")
+	registerSignedOrderField(description, "takerFeeAssetData")
+	registerSignedOrderField(description, "takerFee")
+	registerSignedOrderField(description, "expirationTimeSeconds")
+	registerSignedOrderField(description, "salt")
+	registerSignedOrderField(description, "signature")
+}
+
+func registerStatsTest(description string) {
+	registerStatsField(description, "version")
+	registerStatsField(description, "pubSubTopic")
+	registerStatsField(description, "rendezvous")
+	registerStatsField(description, "peerID")
+	registerStatsField(description, "ethereumChainID")
+	registerStatsField(description, "latestBlock | hash")
+	registerStatsField(description, "latestBlock | number")
+	registerStatsField(description, "numOrders")
+	registerStatsField(description, "numPeers")
+	registerStatsField(description, "numOrdersIncludingRemoved")
+	registerStatsField(description, "numPinnedOrders")
+	registerStatsField(description, "maxExpirationTime")
+	registerStatsField(description, "startOfCurrentUTCDay")
+	registerStatsField(description, "ethRPCRequestsSentInCurrentUTCDay")
+	registerStatsField(description, "ethRPCRateLimitExpiredRequests")
+}
+
+// FIXME(jalextowle): Generalize for non-empty validation results
```

Comment:
Is this a "nice to have" feature, or something you plan to address in a subsequent PR?

Reply:
This is a dangling comment. I left this for myself to make sure that I implemented the for-loops in the `registerValidationResultsTest` function. I will remove.

Reply:
This is a dangling comment. I left this for myself to make sure that I implemented the for-loops in the registerValidationResultsTest function. I will remove.